p2p/transport/webtransport/transport.go

```diff
@@ -264,6 +265,9 @@ func (t *transport) upgrade(ctx context.Context, sess *webtransport.Session, p p
 	if err != nil {
 		return nil, err
 	}
+	if err = c.Close(); err != nil {
```

Comment:
Do we care about this error at all? Can we do `defer c.Close()` instead?

Reply:
I made that change.